doc/release/upcoming_changes/29137.compatibility.rst

```diff
@@ -0,0 +1,5 @@
+* For integer arrays, ``numpy.round`` function now returns a copy of input. Previously,
+  it returned a view for integer inputs and a copy for floating-point inputs. The change
+  brings ``round`` in line with ``ceil``, ``floor`` and ``trunc``--- all of which always
+  return a copy. 
```

Comment:
```suggestion
* `numpy.round` now always returns a copy. Previously, it returned a view
  for integer inputs for ``decimals >= 0`` and a copy in all other cases.
  This change brings ``round`` in line with ``ceil``, ``floor`` and ``trunc``. 
```
I dunno, you can also keep what you had. Just started editing to remove function and make it a link and then this happened :).